pkg/kubelet/pod_workers.go

```diff
@@ -291,9 +291,9 @@ func killPodNow(podWorkers PodWorkers, recorder record.EventRecorder) eviction.K
 		}
 
 		// we timeout and return an error if we don't get a callback within a reasonable time.
-		// the default timeout is relative to the grace period (we settle on 2s to wait for kubelet->runtime traffic to complete in sigkill)
+		// the default timeout is relative to the grace period (we settle on 10s to wait for kubelet->runtime traffic to complete in sigkill)
 		timeout := int64(gracePeriod + (gracePeriod / 2))
-		minTimeout := int64(2)
+		minTimeout := int64(10)
```

Comment:
My only concern is that somehow due to some issues, Kubelet hit this timeout upon OUT-OF-MEMORY issue, we failed to kill another pod / container within a reasonable time, thus leading to a sys OOM. We know kernel once a while run into a unkillable process issue. :-) 

But on another side, I knew 2s is not acceptable. I am ok to take this as is.